api/lib/infrastructure/repositories/sco-certification-candidate-repository.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
       .from('certification-candidates')
       .join('schooling-registrations', 'schooling-registrations.id', 'certification-candidates.schoolingRegistrationId')
       .where('schooling-registrations.organizationId', '=', organizationId)
+      .where('schooling-registrations.isDisabled', '=', false)
```

Comment:
On pourrait regrouper les deux
```suggestion
      .where({
        'schooling-registrations.organizationId': organizationId,
        'schooling-registrations.isDisabled': false
      })
```